apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -1283,7 +1283,7 @@
       "noSearchResults": "Sorry, we did not find any search results.",
       "retrySearchKeyword": "Please retry the search with another keyword.",
       "retrySearchParams": "Please retry the search with another parameters.",
-      "trackFavAssets": "Track your favourite",
+      "trackFavAssets": "Track your favorite",
```

Comment:
I think both are fine :D American vs british english

Reply:
![](https://media.giphy.com/media/12DB9RMTtvGvYY/giphy.gif)

Reply:
We settled for american for all